src/components/picker/NativePicker.js

```diff
@@ -7,6 +7,7 @@ import TouchableOpacity from '../touchableOpacity';
 import {Colors} from '../../style';
 import {WheelPicker} from '../../incubator';
 
+// TODO: remove this file?
```

Comment:
Why this is related to the reanimated upgrade?

Reply:
It uses `Incubator.WheelPicker` so I wanted to test it, but nothing is currently using this file; I did not want to make this PR any larger or risk anything so I've added a TODO.